### PR TITLE
#1 add test for GPT parser

### PR DESCRIPTION
**Pull request by flopitta90**
could you please check it?

### CODE REVIEW

File: my-vocabulary/src/components/WordCard.jsx [hunk 1]
```diff
@@ -53,17 +54,9 @@ export const WordCard = () => {
     }
     else navigate('/login')
   }
-  const indexWord = text?.text?.indexOf('Meaning')
-  const indexMeaning = text?.text?.indexOf('Sentences:')
-  const indexSentences = text?.text?.indexOf('1')
-  const indexSentence1 = text?.text?.indexOf('2')
-  const indexSentence2 = text?.text?.indexOf('3')
+  console.log(text)
```
Comment (nicomfe) [inline my-vocabulary/src/components/WordCard.jsx:57]: please remove this console.log

File: my-vocabulary/src/functions/__test__/parseGPT.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { parseGPT } from "../parseGPT";
+// import {describe, expect, test} from '@jest/globals';
```
Comment (nicomfe) [inline my-vocabulary/src/functions/__test__/parseGPT.test.js:2]: please remove this commented line.

File: my-vocabulary/src/functions/parseGPT.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+export const parseGPT = (text) => {
```
Comment (nicomfe) [inline my-vocabulary/src/functions/parseGPT.js:1]: I would call this method parseChatGptResponse instead.

**Review by nicomfe — CHANGES_REQUESTED** [review]

File: my-vocabulary/src/components/WordCard.jsx (hunk 1 above)
Comment (flopitta90) [reply my-vocabulary/src/components/WordCard.jsx]: done

File: my-vocabulary/src/functions/__test__/parseGPT.test.js (hunk 2 above)
Comment (flopitta90) [reply my-vocabulary/src/functions/__test__/parseGPT.test.js]: done

File: my-vocabulary/src/functions/parseGPT.js (hunk 3 above)
Comment (flopitta90) [reply my-vocabulary/src/functions/parseGPT.js:1]: It is not chat gpt though.. I could add the response

**Pull request merged**